pom.xml

```diff
@@ -85,7 +85,7 @@
     </modules>
 
     <properties>
-        <open-metadata.version>2.9-SNAPSHOT</open-metadata.version>
+        <open-metadata.version>2.10-SNAPSHOT</open-metadata.version>
         <connector.version>2.9-SNAPSHOT</connector.version>
```

Comment:
Since we're bumping the Egeria core version on which we depend, please also bump the version of the connector itself to be the same (2.10-SNAPSHOT) -- this makes it simpler for consumers to understand the minimal version of Egeria that will be required to run this version of the connector.

Reply:
I forgot that releasing for this repo currently has other blockers that we're waiting on, so will merge this for now and handle the version updates separately (before any subsequent release).